versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractReferenceNotFound.java

```diff
@@ -76,6 +76,7 @@ void run(VersionStore<BaseContent, CommitMessage, BaseContent.Type> store)
     }
   }
 
+  @SuppressWarnings({"StreamResourceLeak", "MustBeClosedChecker"})
```

Comment:
does this really need `StreamResourceLeak` ?
according to https://errorprone.info/bugpattern/StreamResourceLeak this is only for using the `Files` class.

regarding `MustBeClosedChecker` we should comment on when it is objectively not needed versus we are skipping it due to convenience (i assume that is what we do here, since these are for tests?)